repository/pom.xml

```diff
@@ -15,7 +15,7 @@
   </parent>
 
   <artifactId>repository</artifactId>
-  <version>4.2.0-SNAPSHOT</version>
+  <version>4.4.0-SNAPSHOT</version>
```

Comment:
Shouldn't this be inherited from the parent `pom.xml`?  That has the magic value BUILD_VERSION and I can't remember where that comes from.